### ISSUE TITLE
Unique icons for Android share button actions

### ISSUE THREAD

**Issue by ThubLives** [xbmc/Kore#748]
## Feature suggestion
**Detailed Description**
Unique icons for the Android "share" button should help to differentiate them on lists that only show the icon.

### Context
When handling a list of YouTube links, I often want to start playing one video, then queue a number of other videos.  Most Android implementations seem to show only a few recent actions in the initial share pop-up, but only as icons.  To see the names of these actions, the user must open the secondary list which shows all available actions/apps and their names.

On my phone, it only shows two recent actions and if they both refer to Kore (one is probably Play on Kodi and the other Queue on Kodi) then they may switch positions and as I'm adding to the queue, I'll choose the wrong one and the queue will be wiped out when I unknowingly chose to play instead of queue it.

### Possible Implementation
I could make a suggestion for the icon design:  Play on Kodi could use the Kore logo with the rightmost rounded square replaced by a "play" triangle, and Queue on Kodi would similarly replace the rightmost rounded square with a plus sign.

### Additional context, screenshots or links
Here are some relevant links or screenshots:

An example of the confusion:  
![Screenshot_20200622-014825_resized](https://user-images.githubusercontent.com/13872430/85264202-91bf9000-b42d-11ea-97cd-e2b5f6284158.jpg)

A rough concept for the Pay on Kodi action:  ![Kore_PlayonKodi_Actino_Icon_Shadow_512](https://user-images.githubusercontent.com/13872430/85261442-65a21000-b429-11ea-9e4c-378e69e80358.png)

*Note: Team Kodi will consider this item however we will not make any promises if it will be included.*

**Comment by poisdeux** [comment]
Good idea. It would be nice to have separate icons for queueing and playing. If you or someone else can come up with some nice designs for the queue and the play share buttons I can implement it.

**Comment by corylowry12** [comment]
I second this. Its not a huge issue, but it would definitely be more convenient.
